plugins/postgres/dbt/adapters/postgres/relation.py

```diff
@@ -20,3 +34,57 @@ def __post_init__(self):
 
     def relation_max_name_length(self):
         return 63
+
+    def get_materialized_view_config_change_collection(
+        self, relation_results: RelationResults, runtime_config: RuntimeConfigObject
+    ) -> Optional[PostgresMaterializedViewConfigChangeCollection]:
+        """
+        Postgres-specific implementation of `BaseRelation.get_materialized_view_config_changes`.
+
+        The only tracked changes for materialized views are indexes.
+        """
+        config_change_collection = PostgresMaterializedViewConfigChangeCollection()
+
+        existing_materialized_view = PostgresMaterializedViewConfig.from_relation_results(
+            relation_results
+        )
+        new_materialized_view = PostgresMaterializedViewConfig.from_model_node(
+            runtime_config.model
+        )
+
+        if index_config_changes := self._get_index_config_changes(
```

Comment:
ooooh we can use the walrus operator now because of python 3.8+ support! :=)

Reply:
Yup, and I sprinkled it everywhere. We do that operation really often, so I'm excited now.